botocore/httpsession.py

```diff
@@ -224,6 +247,26 @@ def _setup_ssl_cert(self, conn, url, verify):
             conn.cert_reqs = 'CERT_NONE'
             conn.ca_certs = None
 
+    def _setup_proxy_ssl_context(self, proxies_settings):
+        proxy_ca_bundle = proxies_settings.get('proxy_ca_bundle')
+        proxy_cert = proxies_settings.get('proxy_client_cert')
+        if proxy_ca_bundle is None and proxy_cert is None:
+            return None
```

Comment:
What's the behavior around creating a proxy manager with an SSL context of `None`?

Reply:
That's the default arg for urllib3, but we actually strip the entire kwarg in `_proxies_kwargs` before calling urllib3, so this is a no-op.